pgmpy/factors/continuous/LinearGaussianCPD.py

```diff
@@ -125,3 +125,36 @@ def __str__(self):
     def __repr__(self):
         str_repr = self.__str__()
         return f"<LinearGaussianCPD: {str_repr} at {hex(id(self))}"
+
+    @staticmethod
+    def get_random(variable, evidence, loc=0.0, scale=1.0, seed=None):
+        """
+        Generates a LinearGaussianCPD instance with random values on `variable` with
+        parents/evidence `evidence` with beta and std sampled from loc and scale
+
+        Parameters
+        ----------
+        variable: str, int or any hashable python object.
```

Comment:
Could you please add the other arguments to the docstring? And an example would be nice too.

Reply:
Right. Fixed in c37e32ef39296ed8cd95328d0242a6d6d415774f